dispatch/modules/content/models.py

```diff
@@ -451,10 +451,33 @@ def get_absolute_url(self):
         """ Returns page URL. """
         return "%s%s/" % (settings.BASE_URL, self.slug)
 
-class Video(Model):
+class Video(Model, AuthorMixin):
     title = CharField(max_length=255)
     url = CharField(max_length=500)
 
+    authors = ManyToManyField(Author, related_name='video_authors')
+    tags = ManyToManyField('Tag')
+
+    created_at = DateTimeField(auto_now_add=True)
+    updated_at = DateTimeField(auto_now=True)
+
+    AuthorModel = Author
+
+    # Overriding
+    def save(self, **kwargs):
```

Comment:
We aren't doing anything special in `save()` (yet) so we can remove this method

Reply:
Good catch! I forgot I removed the custom save operations